src/EditorFeatures/Core/SymbolSearch/SymbolSearchUpdateEngine.Update.cs

```diff
@@ -96,7 +98,7 @@ public Updater(SymbolSearchUpdateEngine service, string source, string localSett
                 _source = source;
 
                 _cacheDirectoryInfo = new DirectoryInfo(Path.Combine(
-                    localSettingsDirectory, "PackageCache", string.Format(Invariant($"Format{AddReferenceDatabase.TextFileFormatVersion}"))));
+                    localSettingsDirectory, "PackageCache", string.Format(Invariant($"Format{AddReferenceDatabaseTextFileFormatVersion}"))));
```

Comment:
this somewhat defeats the purpose.  if Elfie changes we want to now access patchfiles with hte new version number.  this would allow us to generate patches for older and newer versions of VS compatibly (since they would live in separate files).

Reply:
> if Elfie changes

Elfie hasn't, doesn't, and won't change.

Reply:
Why would it not change?  We're literally talking to nuget now about potentially changing the format here to be easier to produce and ingest.

Reply:
Regardless, we don't want to define it own constant here. It needs to be based off the API we're using to read/patch the data. This is so that the library we're using to read, and the server creating the data can agree on what format is being used among many clients existing at the same time

Reply:
It seems we are only using the version to fetch database/patch from the server, and we always download the v1 files. e.g.

```cs
$"Elfie_V{AddReferenceDatabase.TextFileFormatVersion}/Latest.xml"
```